src/components/building-information/building-information.component.tsx

```diff
@@ -68,112 +76,67 @@ class BuildingInformation extends React.Component<IProps, IState> {
-                                <Text style={styles.normalText}>
-                                  {"\u2022  "}
-                                </Text>
-                                <Text
-                                  nativeID="departments"
-                                  style={styles.linkText}
-                                  onPress={() => {
-                                    if (department.link) {
-                                      Linking.openURL(department.link);
-                                    }
-                                  }}
-                                >
-                                  {department.title}
-                                </Text>
-                              </View>
-                            );
-                          })}
-
-                        <View testID="scrollServices">
-                          {building.services.length !== 0 ? (
-                            <Text style={styles.headerText}>Services:</Text>
-                          ) : (
-                            <Text style={styles.headerText}>
-                              No services available.
-                            </Text>
-                          )}
-
-                          {building.services &&
-                            building.services.map((service) => {
-                              return (
-                                <View key={service.id} style={styles.listItem}>
-                                  <Text style={styles.normalText}>
-                                    {"\u2022  "}
-                                  </Text>
-                                  <Text
-                                    testID={service.title}
-                                    nativeID="services"
-                                    style={styles.linkText}
-                                    onPress={() => {
-                                      if (service.link) {
-                                        Linking.openURL(service.link);
-                                      }
-                                    }}
-                                  >
-                                    {service.title}
-                                  </Text>
-                                </View>
-                              );
-                            })}
-                        </View>
-                      </View>
-                    </ScrollView>
-                  </View>
-                );
-              }
-            )}
+
+            <View testID="panel" style={styles.panelContent}>
+              <View style={styles.uparrow}>
+                <AntDesign name="up" size={40} />
+              </View>
+              <Text
+                testID="panelDisplayName"
+                style={{
+                  ...styles.buildingName,
+                  fontSize: building.displayName.length > 30 ? 18 : 24,
+                }}
```

Comment:
Not a blocker but is there a reason you're not using flatten here? You've specifically requested that it be used over this method, does it not work here?

Reply:
true, i gotta do what I preach.